docs/changelog.md

```diff
@@ -10,11 +81,47 @@
 
     The `DetectionDataset.images` property is deprecated and will be removed in `supervision-0.26.0`. Please loop over images with `for path, image, annotation in dataset:`, as that does not require loading all images into memory.
```

Comment:
I'd move info about those deprecations higher up, to section talking about `DetectionDataset`.